README.md

```diff
@@ -14,6 +14,10 @@ A curated list of papers and open-source resources related to DUSt3R, an emergin
 
 <br>
```

Comment:
you can remove the "\<br\>" tag between subfields [Gaussian Splatting] and [Robotics], and then I merge the request. Thanks!

Reply:
Just removed the <br> there!